src/panel/service_panel.js

```diff
@@ -1,47 +1,81 @@
-import ServicePanelView from '../views/service_panel.dot';
-import Panel from '../libs/panel';
-import nconf from '@qwant/nconf-getter';
+/* globals _ */
+import React from 'react';
+import ReactDOM from 'react-dom';
```

Comment:
For now we need `react-dom` too because we render this component in the DOM in this file.
Later, when the migration is finished, we'll have only one ReactDOM call at the app top level, every other component will be purely declarative.